### PR TITLE
[INLONG-619]Configuration for Go SDK.

### PR DESCRIPTION
**Pull request by TszKitLo40**
Signed-off-by: Zijie Lu <wslzj40@gmail.com>

### CODE REVIEW

**Comment by TszKitLo40** [comment]
@gosonzhang @charlely PTAL

File: tubemq-client-twins/tubemq-client-go/config/config.go [hunk 1]
```diff
@@ -19,14 +19,184 @@
...
+			TLSKeyFile string
+			// TTSServerName for TLS.
+			TLSServerName string
+		}
 	}
+
+	// Consumer is the namespace for configuration related to consume messages,
+	// used by the consumer
+	Consumer struct {
+		masters []string
```
Comment (charlely) [inline tubemq-client-twins/tubemq-client-go/config/config.go:53]: How do external packages use these non-exported parameters?

File: tubemq-client-twins/tubemq-client-go/config/config.go [hunk 2]
```diff
@@ -19,14 +19,184 @@
...
+			// TLSKeyFile for TLS.
+			TLSKeyFile string
+			// TTSServerName for TLS.
+			TLSServerName string
+		}
 	}
+
+	// Consumer is the namespace for configuration related to consume messages,
+	// used by the consumer
+	Consumer struct {
```
Comment (charlely) [inline tubemq-client-twins/tubemq-client-go/config/config.go:52]: Need to add a note.

**Review by charlely — APPROVED** [review]
LGTM

**Comment by gosonzhang** [comment]
+1

**Pull request merged**